lib/types.ts

```diff
@@ -29,6 +29,7 @@ export interface DocumentWithLinksAndLinkCountAndViewCount extends Document {
     versions: number;
   };
   links: Link[];
+  isArchived: boolean;
```

Comment:
this is not necessary because the custom interface already extends `Document` so isArchived will be part of the interface by default